test/extended/util/test.go

```diff
@@ -400,7 +400,7 @@ var (
 			`ClusterDns \[Feature:Example\] should create pod that uses dns`,             // https://bugzilla.redhat.com/show_bug.cgi?id=1711601
 			`should be rejected when no endpoints exist`,                                 // https://bugzilla.redhat.com/show_bug.cgi?id=1711605
 			`PreemptionExecutionPath runs ReplicaSets to verify preemption running path`, // https://bugzilla.redhat.com/show_bug.cgi?id=1711606
-			`TaintBasedEvictions`,                                                        // https://bugzilla.redhat.com/show_bug.cgi?id=1711608
+			`EquivalenceCache`,    // Equivalence tests are planned to be removed in 1.15, so we can safely disable them.
```

Comment:
The upstream PR is tagged for merge, @damemi mind picking this up and just adding the `EquivalenceCache` test in the PR, the other test is handled in a separate PR, iirc.

Reply:
Spoke to @ravisantoshgudimetla earlier this week and he said he will take care of this. Ravi, do you still want to pick this back up?

Reply:
Ya, I will send a PR today